src/milind/PrimeNumberFrom2To100.java

```diff
@@ -0,0 +1,28 @@
+package milind;
+
+public class PrimeNumberFrom2To100 {
+	
+	void primeNumber(int rangeStart, int rangeEnd) {
+		System.out.println("Prime Number : " );
+		for(int index = rangeStart; index <= rangeEnd; index++) {
+			int count = 0;
+			for(int innerIndex = rangeStart; innerIndex <= index/2; innerIndex++) {
+				if(index % innerIndex == 0) {
+					count++;
+				}

```

Comment:
once this if condition satisfied then number is not prime number then you can break the loop, why we need to iterate till endpoint right